lib/Watching.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * @file
+ * @copyright  2016 Matt Smith (Development Seed)
+ * @license    Licensed under {@link https://spdx.org/licenses/BSD-3-Clause-Clear.html BSD-3-Clause-Clear}.
+ *             Github.js is freely distributable.
+ */
+
+import Requestable from './Requestable';
+import debug from 'debug';
+const log = debug('github:watching');
+
+/**
+ * Watching a Repository registers the user to receive notifications on new discussions, as well as events in the user's
+ * activity feed.
+ */
+export default class Watching extends Requestable {
```

Comment:
I didn't know there was such a plugin. I forget the exact reason that we're using `module.exports` but it boiled down to nothing works if you use the `export default` syntax.

If there's a plugin that fixes things while allowing us to use the new syntax then I'm all for it.